add codespace support (#1726)
Azure/azure-sdk-for-c: sdk/samples/iot/README.md

To view scenario-focused examples using the API calls, please view the Azure IoT Client [introductory examples](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/docs/iot/README.md#examples). General [coding patterns](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/docs/iot/coding_patterns.md) that are MQTT stack agnostic are also available to view.

## Github Codespaces

> You can use Github Codespaces to be up and running in less than two minutes! Here are the steps to follow (assuming you already have an IoT Hub set up).

Reviewer: Nit, avoid saying minutes in case things are running slow 
```suggestion
You can use Github Codespaces to be up and running quickly! Here are the steps to follow (assuming you already have an IoT Hub set up).
```
